crates/holochain_integrity_types/src/chain.rs

```diff
@@ -131,47 +141,89 @@ impl<H: Eq + Ord + std::hash::Hash> ChainFilter<H> {
         self
     }
 
+    /// Take all actions until this timestamp is passed.
```

Comment:
Based on this doc comment, I'm still not clear what the boundary condition is. This is all an app developer would get when setting up a filter so this needs to be perfectly expliit about what condition is applied after using this function to build your chain filter.